q2_types/feature_table/_type.py

```diff
@@ -22,11 +22,19 @@
 PresenceAbsence = SemanticType('PresenceAbsence',
                                variant_of=FeatureTable.field['content'])
 
+Composition = SemanticType('Composition',
+                           variant_of=FeatureTable.field['content'])
+
+Balance = SemanticType('Balance',
+                       variant_of=FeatureTable.field['content'])
+
+plugin.register_semantic_types(Composition, Balance)
```

Comment:
I would just append these to the `register_semantic_types` on the next line down